File: Selenium/Activities/Activity_16.java
import java.util.List;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.Select;
public class Activity_16 {
	public static void main(String [] args) {
		WebDriver driver=new FirefoxDriver();
		driver.get("https://training-support.net/webelements/selects");
		System.out.println("title of the page: "+driver.getTitle());
		
		WebElement options=driver.findElement(By.cssSelector("select.h-10"));
		
		Select singleSelect = new Select(options);
		singleSelect.selectByVisibleText("Two");
		System.out.println("second option: "+singleSelect.getFirstSelectedOption().getText());
		singleSelect.selectByIndex(3);
		System.out.println("third option: "+singleSelect.getFirstSelectedOption().getText());
		singleSelect.selectByValue("four");
		System.out.println("fourth option: "+singleSelect.getFirstSelectedOption().getText());
		List<WebElement> alloptions= singleSelect.getOptions();
		System.out.println("All available options:");
		for(WebElement getoption : alloptions)
			System.out.println(getoption.getText());
		driver.quit();
	}
}

File: Selenium/Activities/Activity-16.py
import selenium
from selenium import webdriver
from selenium.webdriver.support.select import Select
from selenium.webdriver.common.by import By
with webdriver.Firefox() as driver:
	driver.get("https://training-support.net/webelements/selects")
	print("title of the page: ",driver.title)
		
	options=driver.find_element(By.CSS_SELECTOR,"select.h-10")
		
	singleSelect =Select(options)
	singleSelect.select_by_visible_text("Two")
	
	print("second option: "+singleSelect.first_selected_option.text)
	singleSelect.select_by_index(3)
	print("third option: "+singleSelect.first_selected_option.text)
	singleSelect.select_by_value("four")
	print("fourth option: "+singleSelect.first_selected_option.text)
	alloptions= singleSelect.options
	print("All available options:")
	for option in alloptions:
		print(option.text)
	driver.quit()